File: backend/src/main/java/ru/chsergeig/shoppy/dao/OrderRepository.java
package ru.chsergeig.shoppy.dao;

import org.jooq.Configuration;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Repository;
import ru.chsergeig.shoppy.jooq.tables.daos.OrdersDao;

@Repository
public class OrderRepository
        extends OrdersDao {

    public OrderRepository(@Autowired Configuration configuration) {
        super(configuration);
    }

}

File: backend/src/main/kotlin/ru/chsergeig/shoppy/dao/OrderRepository.kt
package ru.chsergeig.shoppy.dao

import org.jooq.DAO
import ru.chsergeig.shoppy.jooq.tables.pojos.Orders
import ru.chsergeig.shoppy.jooq.tables.records.OrdersRecord

interface OrderRepository : DAO<OrdersRecord, Orders, Int>
